higlass/server.py

```diff
@@ -256,48 +264,59 @@ def setup(self):
         disk_cache_size = 2 ** 25
         disk_cache_dir = self.diskcache_directory
         lru_capacity = 400
-        print(
-            "self.diskcache_directory",
-            self.diskcache_directory,
-            op.exists(self.diskcache_directory),
-        )
+        # print(
+        #     "self.diskcache_directory",
+        #     self.diskcache_directory,
+        #     op.exists(self.diskcache_directory),
+        # )
 
         def start_fuse(directory, protocol):
-            print("starting fuse")
-            fuse = FUSE(
-                HttpFs(
-                    protocol,
-                    disk_cache_size=disk_cache_size,
-                    disk_cache_dir=self.diskcache_directory,
-                    lru_capacity=lru_capacity,
-                ),
-                directory,
-                foreground=False,
-                allow_other=True
-            )
-        proc1 = mp.Process(target=start_fuse, args=[self.http_directory, 'http'])
+            # print("starting fuse")
+            try:
+                fuse = FUSE(
+                    HttpFs(
+                        protocol,
+                        disk_cache_size=disk_cache_size,
+                        disk_cache_dir=self.diskcache_directory,
+                        lru_capacity=lru_capacity,
+                    ),
+                    directory,
+                    foreground=False,
+                    allow_other=True
+                )
+            except RuntimeError as e:
+                if str(e) != "1":
+                    raise e
```

Comment:
😂

But shouldn't we get some indication of the "1" error to let us know that FUSE didn't start?

Reply:
Well, so the "1" error doesn't mean it didn't start. In fact, it always started for me but still logged out this "1" error twice which took up at least 256px of vertical space. I tried to find out what this stupid error message is supposed to tell us but I couldn't find anything.

Reply:
Sometimes errors get redirected to Jupyter's server log...

Reply:
This should be logged in the future